<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Variogram_Map</class>
 <widget class="QDialog" name="Variogram_Map">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>739</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mapa de Variogramas</string>
  </property>
  <property name="windowIcon">
   <iconset resource="imagens.qrc">
    <normaloff>:/imagens/SimpleVar.png</normaloff>:/imagens/SimpleVar.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Variáveis</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Propriedade da cabeça</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="Propr_cabeca"/>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Propriedade do rabo</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="Prop_rabo"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Direção do Norte do plano considerado</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Azimute </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="Azimute">
        <property name="minimum">
         <number>-360</number>
        </property>
        <property name="maximum">
         <number>360</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Mergulho</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="Mergulho">
        <property name="minimum">
         <number>-360</number>
        </property>
        <property name="maximum">
         <number>360</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Parâmetros do mapa de variogramas</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="5">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Min npares         </string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Tamanho  lag</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>n lags</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QSpinBox" name="Lag_size">
        <property name="maximum">
         <number>999999</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="nLag">
        <property name="maximum">
         <number>999999</number>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QSpinBox" name="Min_npares">
        <property name="maximum">
         <number>99999999</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>GroupBox</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Número de células </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="Ncelulas">
        <property name="minimum">
         <number>5</number>
        </property>
        <property name="maximum">
         <number>2000</number>
        </property>
        <property name="value">
         <number>50</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Grau de refinamento</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="Grau_refinamento">
        <property name="minimum">
         <number>10</number>
        </property>
        <property name="maximum">
         <number>350</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Escala de Cor</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="Gradient_type">
        <item>
         <property name="text">
          <string>Grayscale</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Hot</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Cold</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Polar</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Hues</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Spectrum</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Calcular Mapa de Variogramas</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCustomPlot" name="customPlot" native="true">
     <property name="minimumSize">
      <size>
       <width>350</width>
       <height>350</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>Salvar Figura</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="imagens.qrc"/>
 </resources>
 <connections/>
</ui>
